cni/network/network_test.go

```diff
@@ -76,7 +76,7 @@ func GetTestResources() *NetPlugin {
 	plugin.report = &telemetry.CNIReport{}
 	mockNetworkManager := acnnetwork.NewMockNetworkmanager()
 	plugin.nm = mockNetworkManager
-	plugin.ipamInvoker = NewMockIpamInvoker(false, false, false)
+	plugin.ipamInvoker = NewMockIpamInvoker(true, false, false) // enable ipv6 flag for dualstack test cases
```

Comment:
I believe this shouldn't be just set to true for all the tests.
you should update 
func GetTestResources() *NetPlugin { to receive a flag for ipv6 and then pass it tp NewMockIpamInvoker